src/deps/uws.zig

```diff
@@ -2413,6 +2413,12 @@ pub const UVLoop = extern struct {
     pre: *uv.uv_prepare_t,
     check: *uv.uv_check_t,
 
+    pub fn init() *UVLoop {
+        return uws_get_loop_with_native(bun.windows.libuv.Loop.get());
```

Comment:
is libuv's default loop threadlocal or global? If it's threadlocal, this is fine. if it's not, we need to change it